__device-tests__/gutenberg-editor-block-insertion.test.js

```diff
@@ -100,6 +102,7 @@ describe( 'Gutenberg Editor tests for Block insertion', () => {
 			await editorPage.dismissKeyboard();
 		}
 
+		await swipeDown( driver );
```

Comment:
@SergioEstevao , do you happen to know why do we need this operation here?

Reply:
This was @JavonDavis work, so I'm not sure about the rational of this.